src/runtime/auto-prompt-manager.js

```diff
@@ -730,18 +732,22 @@ export class AutoPromptManager {
    * [t1]         NO  (free read)   YES
    * [t1, t2]     NO  (free read)   YES
    * [t1, t2, t3] YES (2nd prompt)  NO
+   * @param {number|undefined} expConfigNumFreeReads
    * @return {!Promise<boolean>}
    */
-  async secondPromptDelayExperimentSuppressesPrompt_() {
-    const numFreeReads = 2; // (b/267650049) 2 free reads
-    const shouldShowAutopromptTimestamps = await this.storage_.getEvent(
+  async secondPromptDelayExperimentSuppressesPrompt_(expConfigNumFreeReads) {
+    const numFreeReads =
+      expConfigNumFreeReads && expConfigNumFreeReads > 0
+        ? expConfigNumFreeReads
+        : 2; // (b/267650049) default 2 free reads
```

Comment:
Last PR, we explicitly removed this param to simplify this function, but here we need to read this from `article`.

Reply:
see https://github.com/subscriptions-project/swg-js/pull/2763#discussion_r1119167308